frontend/common/backendUrl.js

```diff
@@ -1,4 +1,4 @@
-// const backendURL  = 'http://localhost:3000' // local environment
+// const backendURL  = 'http://localhost:4000' // local environment
```

Comment:
Revert this as well as it was before.
Also, note to revert all the config changes back to as it was before pushing to the PR.